### DISCUSSION TITLE
windows防火墙烦死了

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#1469] [提问] [answered]
这个windows defender总是拦我最新的更新操作，我的win10咋关了它，或者咋添加信任项

**Comment by WorldHim** [comment]
可以完全卸载 defender，然后换其他杀毒软件（不推荐

**Comment by WForst-Breeze** [comment]
不应该出现可以打开 Windows 安全中心 而无法找到 添加排除项 这件事的按理讲。
1. 您可以完全禁用 Windows Defender 而更换为其它杀软（eg：火绒）但这很有风险，所以我们并不推荐此类做法：[完全禁用 Windows Defender](https://blog.csdn.net/WindSunLike/article/details/105873536)；
3. 如果您仍还想继续使用 Windows Defender，或许可以传个截图看看？正常情况下是这样的：
![image](https://user-images.githubusercontent.com/110760354/226158535-df2c4ee0-ec41-47d8-affd-3cbe0ab1fea8.png)

**Reply by iPodtouch6557** [reply]
![image](https://user-images.githubusercontent.com/118656946/226178074-d30f832d-c280-4729-955a-3e9b19536e96.png)
so
youwillbelikethemtoo
abandonme.

**Comment by iPodtouch6557** [comment]
![image](https://user-images.githubusercontent.com/118656946/226178288-5d93415e-aa9d-4557-b514-6b592d339e0c.png)
youwill问号？

**Reply by WForst-Breeze** [reply]
草……真的没有……我不用联想笔记本，涉及知识盲区了（）抱歉，等等看其他朋友有没有会处理的qwq
（如果你愿意的话，去问问联想客服……？

**Comment by LTCatt** [comment, marked as answer]
推荐使用其他杀毒软件（例如火绒），这会替代 Windows Defender……

**Comment by hejiehao** [comment]
当windows defender检测到其他杀毒软件的时候，windows defender会自动关闭
需要在上述提到的杀毒软件（火绒和迈克菲）信任PCL2

**Comment by Chiloven945** [comment]
下个其他的杀软不就行了吗，我的火绒就没有报毒过PCL
![image](https://user-images.githubusercontent.com/116699482/226364152-6cc3cf04-fe6e-435f-b778-c2733a8b73db.png)
不过我发现defender还是会自动扫描，上次我PCL被删了好几次（

**Reply by Chiloven945** [reply]
等一下被识别成病毒怕不是千万别点的东西吧（

**Reply by WForst-Breeze** [reply]
不会的吧……（

**Reply by Chiloven945** [reply]
那种样子的确会像是病毒干的事情

**Reply by iPodtouch6557** [reply]
> 下个其他的杀软不就行了吗，我的火绒就没有报毒过PCL ![image](https://user-images.githubusercontent.com/116699482/226364152-6cc3cf04-fe6e-435f-b778-c2733a8b73db.png) 不过我发现defender还是会自动扫描，上次我PCL被删了好几次（）

其实我在 #666 遇到的问题跟这个实质上是一样的，但是我就解决不了它（）
另外补充，在文件离奇消失时我这边的错误截图不是“锟斤铐烫烫烫”，而是什么“没有足够权限这一类的”，下载更新就是防火墙拦截，弹窗提示的很诚实（笑）

**Comment by iPodtouch6557** [comment]
好的，现在我在本地组里关了windefender，还是会拦截
我现在都要怀疑不是windefender干的事了（）

**Reply by HamsterYrc** [reply]
怕不是迈克菲干的好逝

**Reply by iPodtouch6557** [reply]
> 怕不是迈克菲干的好逝

迈克菲打不开，我必须绕开它，但是现在调查结果是97%的可能是他的事（）
要么就卸了他，但这样做风险比关了windefender还大，我不敢:(     哪怕卸了再下回来

**Reply by WForst-Breeze** [reply]
直接换火绒或许会安全且直接一点……（

**Reply by iPodtouch6557** [reply]
> 直接换火绒或许会安全且直接一点……（

下了，没用

**Comment by iPodtouch6557** [comment]
谢谢大家，这问题突然解决了（）现在有了最新版了
